packages/signaldb/src/SyncManager/index.ts

```diff
@@ -165,7 +160,11 @@ export default class SyncManager<
         type: 'insert',
         data: item,
       })
-      this.schedulePush(options.name)
+
+      const isSyncing = this.getCollection(options.name)[2]
```

Comment:
To avoid ambiguity with the `isSyncing()` method of the class, I would prefer change this maybe to `isPaused` and negate the logic.